dace/libraries/linalg/__init__.py

```diff
@@ -1,5 +1,4 @@
 # Copyright 2019-2023 ETH Zurich and the DaCe authors. All rights reserved.
 from dace.library import register_library
-from .nodes import *
```

Comment:
you should not remove this statement from a library init, we use it for querying nodes

Reply:
I see I will reintroduce it. I had a circular import issue with them and looked unnecessary in the first glance. 
I will reintroduce them and handle the circular import differently.